src/Components/EventContainer.tsx

```diff
@@ -1,23 +1,19 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useContext } from 'react';
 import { useParams } from 'react-router-dom'
 import '../Styles/EventContainer.css';
 import apiCalls from '../utilities/apiCalls';
 import Opp from './Opp'; 
+import { EventContext } from '../Context/EventContext';
 import { Event } from '../utilities/Types';
 
 
-const EventContainer = () => {
 
-  const [events, setEvents] = useState<Event[] | []>([])
-  const zip = useParams().zipcode
-  console.log(events)
+const EventContainer = () => {
   
-  useEffect(() => {
-    apiCalls.loadAllEvents()
-    .then(data => setEvents(data.data))
-  }, [])
+  const { events, category, filteredEvents, filterByCategory } = useContext(EventContext)
+  const zip = useParams().zipcode
   
-  const eventCards = events && events.map((event, index) => {
+  const eventCards = events && events.map((event: any, index: any) => {
```

Comment:
let's make sure we come back and set these as something besides any - we have an Event interface imported into this file that we can use here on line 16.